packages/@react-spectrum/test-utils/src/events.ts

```diff
@@ -98,3 +98,8 @@ export function typeText(el: HTMLElement, value: string, opts?: ITypeOpts) {
     skipClick = true;
   }
 }
+
+export function pressKey(el, opts = {}) {
```

Comment:
heh, nice, i've been meaning to get something like this added for a while, thanks for doing it!